docs/getting-started/Overview.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Overview
+draft: true
+---
+
+# Overview
+
+Welcome to the Identity Documentation!
+
+Here you can find out more about how you can use our software to verify your user's identities, securely validate identities for third-parties, and build dApps that safely manage your user's identities.
+
+Identity validators and identity requesters should start by looking at the Gateway Protocol.
+
+## What is the Gateway Protocol?
+
+The Gateway Protocol allows both identity requesters and identity validators (also known as Gatekeepers) to securely confirm that a person exists and they are the valid owners of an identity.
+
+The Gateway Protocol is permissionless, allowing a wide variety of Gatekeeper networks to act as identity validators for dApps that need identity verification.
+
+Gatekeeper Networks are made up of identity validators who are compensated for performing identity verification tasks for supported dApps. Each Gatekeeper Network has it's own set of rules and requirements which are determined by the Gatekeepers who vote for changes both within their Gatekeeper Network and for the broader Gateway Protocol.
+
+## What is did:sol?
+
+Decentralized identifiers (DIDs) are a tool that allows users to manage their identity on blockchains, did:sol is the official DID method for the Solana blockchain. This means that did:sol:
+
+- Defines how DIDs work on Solana
+- Offers performance benefits to the controller
+- Supports different Solana verification methods
+- Provides an identifier for use both on-chain and off-chain
+
+## What is Cryptid?
+
+Cryptid integrates with Solana, and is geared towards developers working with on-chain dApps. It allows developers to securely interact with transactions using a user's DID. Developers can make it easy for users to sign transactions and prove their identity while keeping their personal information private and secure. This allows developers to build a variety of features including:
```

Comment:
Starting the sentence off with how Cryptid integrates and who it is targeted for is backwards. We should start with what it is, then talk that about what it integrates with and who it is targeted for.